libs/librepcb/common/dialogs/aboutdialog.ui

```diff
@@ -312,7 +314,7 @@
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
-           <property name="placeholderText" stdset="0">
+           <property name="placeholderText">
```

Comment:
This should be reverted since it breaks compatibility with older Qt versions.

Reply:
QtCreator did that automatically... I'll revert.